crates/uv/tests/it/python_upgrade.rs

```diff
@@ -245,66 +243,6 @@ fn python_upgrade_transparent_from_venv_with_minor_request() {
     );
 }
 
-// TODO(john): Add upgrade support for preview bin Python. After upgrade,
-// the bin Python version should be the latest patch.
-#[test]
-fn python_transparent_upgrade_with_preview_installation() {
```

Comment:
I'm removing this test because it's not clear it can be tested this way. In the `python_install_preview` test, we're now checking that under the right conditions, the `bin` link points to the symlink directory-containing path. But there's no way to `--preview` install an outdated patch version to `bin` and also make it upgradeable (since only default and minor version installations will create a transparently upgradeable `bin` link).